### ISSUE TITLE
Running with 0.21.0 yields a "TypeError: Cannot read property 'title' of null" error that is not thrown with 0.20.1

### ISSUE THREAD

**Issue by DavidBiesack** [asyncapi/html-template#191]
#### Describe the bug

Running with 0.21.0 yields a crash that is not thrown with 0.20.1

#### How to Reproduce

```bash
$ ag --force-write --output gen/doc asyncapi.yaml @asyncapi/html-template
```

(source below)

I'm using this from a node project.
Peviously I had done

```bash
$ npm i --save-dev @asyncapi/generator @asyncapi/html-template
$ npx ag --force-write --output gen/doc asyncapi.yaml @asyncapi/html-template 
```

(I had generator 1.6.13)

I saw a more recent version of `@asyncapi/html-template` was available so I tried it

```bash
$ npm i @asyncapi/html-template@latest
```

but now I get an error:

```
$ ag --force-write --output gen/doc @asyncapi/html-template asyncapi.yaml

Something went wrong:
Template render error: (~/dev/asyncapi/async-events-api-def/node_modules/@asyncapi/html-template/template/index.html) [Line 7, Column 57]
  TypeError: Cannot read property 'title' of null
    at Object._prettifyError (~/dev/asyncapi/async-events-api-def/node_modules/nunjucks/src/lib.js:36:11)
    at ~/dev/asyncapi/async-events-api-def/node_modules/nunjucks/src/environment.js:563:19
    at Template.root [as rootRenderFunc] (eval at _compile (~/dev/asyncapi/async-events-api-def/node_modules/nunjucks/src/environment.js:633:18), <anonymous>:58:3)
    at Template.render (~/dev/asyncapi/async-events-api-def/node_modules/nunjucks/src/environment.js:552:10)
    at Environment.renderString (~/dev/asyncapi/async-events-api-def/node_modules/nunjucks/src/environment.js:380:17)
    at ~/dev/asyncapi/async-events-api-def/node_modules/@asyncapi/generator/lib/renderer/nunjucks.js:32:14
    at new Promise (<anonymous>)
    at nunjucksExport.renderNunjucks (~/dev/asyncapi/async-events-api-def/node_modules/@asyncapi/generator/lib/renderer/nunjucks.js:31:10)
    at Generator.renderFile (~/dev/asyncapi/async-events-api-def/node_modules/@asyncapi/generator/lib/generator.js:688:12)
    at Generator.renderAndWriteToFile (~/dev/asyncapi/async-events-api-def/node_modules/@asyncapi/generator/lib/generator.js:602:27)
```

Here is a trimmed down source file that works with 0.20.1 but fails with 0.21.0:

```yaml
asyncapi: 2.0.0
id: 'https://api.apiture.com/apis/async-api-id/apiture-public-events'
info:
  title: Apiture Public Events
  description: Describes the event channels and event messages that the Apiture Open digital banking platform subscribees for client consumption.
  contact:
    name: Apiture
    email: api@apiture.com
    url: 'https://developer.apiture.com'
  license:
    name: Apiture Open API License Agreement
    url: 'https://developer.apiture.com/docs/Apiture-Open-API-License-Agreement.pdf'
  version: 0.2.0
servers:
  sandbox:
    url: 'https://api.sandbox.apiture.com/async-events'
    description: The sandbox environment (open/sandbox)
    protocol: sqs
tags:
  - name: Accounts
    description: Banking Accounts
channels:
  accounts:
    description: Account related events
    subscribe:
      operationId: accounts
      summary: Accounts
      description: Fired after activity on banking accounts
      tags:
        - name: Accounts
      message:
        oneOf:
          - name: accountCreated
            payload:
              $ref: '#/components/schemas/accountCreated'
            title: Account Created
            summary: User (Bank Customer) successfully created (opened) an account
            tags:
              - name: Accounts
components:
  schemas:
    accountCreated:
      title: Account Created
      description: A banking customer successfully opened a new account.
      properties:
        _id:
          description: The unique ID of this event. This is an opaque string.
          type: string
          maxLength: 48
        occurredAt:
          description: 'The [RFC 3339](https://tools.ietf.org/html/rfc3339) date-time (UTC) when the event occurred.'
          type: string
          format: date-time
      example:
        _id: event1234
        occurredAt: '2021-05-05T13:21:39.375Z'
```

(this source renders fine in https://playground.asyncapi.io/ )

#### Expected behavior

generate doc as with 0.20.1

**Comment by magicmatatjahu** [comment]
@DavidBiesack Hello! We fixed problem in `0.21.1` version, Let us know if we created another bugs! 😅

**Comment by DavidBiesack** [comment]
@magicmatatjahu thank you Maciej - I can confirm that this `0.21.1` update works on my full AsyncAPI source.
